app/src/main/java/vn/mbm/phimp/me/Camera2.java

```diff
@@ -950,12 +950,9 @@ public void surfaceChanged(SurfaceHolder holder, int format, int w, int h) {
 		Log.e("Surface","Change");
 		Camera.Parameters parameters = mCamera.getParameters();
 		int state = Camera2.getState();
-		if(state == 1 )
-		{ parameters.setFlashMode(Camera.Parameters.FLASH_MODE_OFF); }
-		else if(state == 2)
-		{ parameters.setFlashMode(Camera.Parameters.FLASH_MODE_AUTO); }
-		else
-		{ parameters.setFlashMode(Camera.Parameters.FLASH_MODE_ON);	}
+		if(state == 1 )	{ parameters.setFlashMode(Camera.Parameters.FLASH_MODE_OFF); }
```

Comment:
Move the if stmts body to another line. If you want you may not use brackets as well.